docs/ai/conceptual/embeddings.md

```diff
@@ -0,0 +1,75 @@
+
+## Use cases for embeddings
+
+This section lists the main use cases for embeddings. LLMs can process text embeddings, but to use other data types you need to include a suitable embedding model, such as **dall-e-3** for image generation.
+
+> [!TIP]
+> In Semantic Kernel, you can [use nested functions](/semantic-kernel/prompts/calling-nested-functions?tabs=Csharp#calling-a-nested-function) to integrate embeddings in your prompts.
+
+### Use your own data to improve completion relevance
+
+Use your own databases to generate embeddings for your data and integrate it with an LLM to make it available for completions. This use of embeddings is called *retrieval-augmented generation*.
+
+### Increase the amount of text you can fit in a prompt
+
+Use embeddings to increase the amount of text you can fit in a prompt, by reducing the number of tokens per word.
+
+For example, suppose you want to include 500 pages of text in a prompt. The number of tokens for that much raw text will exceed the input token limit, making it impossible to directly include in a prompt. You can use embeddings to summarize and break down large amounts of that text into pieces that are small enough to fit in one input, and then assess the similarity of each piece to the entire raw text. Then you can choose a piece that best preserves the semantic meaning of the raw text and use it in your prompt without hitting the token limit.
+
+### Perform text classification, summarization, or translation
+
+Use embeddings to help a model understand the meaning and context of text, and then classify, summarize, or translate that text. For example, you can use embeddings to help models classify texts as positive or negative, spam or not spam, or news or opinion.
+
+### Generate and transcribe audio
+
+Use audio embeddings to process audio files or inputs in your app.
+
+For example, [Speech service](/azure/ai-services/speech-service/) supports a range of audio embeddings, including [speech to text](/azure/ai-services/speech-service/speech-to-text) and [text to speech](/azure/ai-services/speech-service/text-to-speech). You can process audio in real-time or in batches.
+
+### Turn text into images or images into text
+
+Semantic image processing requires image embeddings, which most LLMs can't generate. Use an image-embedding model to create vector embeddings for images. Then you can use those embeddings with an LLM to create or modify images using text or vice versa. For example, you can [use the DALL·E embedding model to generate images](/azure/ai-services/openai/dall-e-quickstart?tabs=dalle3%2Ccommand-line&pivots=programming-language-csharp) such as logos, faces, animals, and landscapes.
```

Comment:
I don't think DALL-E is an embedding model. I think it's just a generation model.

Reply:
I'm not sure what the difference is - I can't find any info that distinguishes the two as separate. I found DALL-E listed on https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#embeddings

Reply:
As far as I can tell, DALL-E includes/leverages a modified CLIP model, which calculates the vectors.

Reply:
Basically, you need dall-e-3 for image<>text transformations using embeddings.